packages/react-ui-core/src/Gmap/OverlayView.js

```diff
@@ -111,18 +116,19 @@ export default class OverlayView extends PureComponent {
     let display = 'none'
 
     if (position) {
-      const projection = this.overlay.getProjection()
-
-      // Projection is undefined if map has not yet initialized
-      if (!projection) return
-
-      const { x, y } = projection.fromLatLngToDivPixel(position)
+      const { x, y } = this.overlay.getProjection().fromLatLngToDivPixel(position)
```

Comment:
Without the above check, this like could throw if `this.overlay.getProjection()` is undefined or null.

Reply:
True